mlrun/serving/routers.py

```diff
@@ -244,7 +254,150 @@ class OperationTypes(str, Enum):
     explain = "explain"
 
 
-class VotingEnsemble(BaseModelRouter):
+class _ParallelRunInterface:
+    def __init__(
+        self, executor_type: Union[ParallelRunnerModes, ExecutorTypes, str], **kwargs
+    ):
+        """
+        Interface for execute parallel runners
+
+        :param executor_type: The desired way for execute parallel run.
+                              Have 3 option :
+                              * array - running one by one
+                              * process - running in separated process
+                              * thread - running in separated threads
```

Comment:
I think those explanations should also be by side of each attribute in ParallelRunnerModes

Reply:
is that on purpose or a typo that for process we execute that in a separated process but for thread we execute in separate threads

Reply:
now looking at the code it seems like its in separate processes as you create  `ProcessPoolExecutor` and pass it max_workers, so please update docs accordingly